client/src/resources/lang/en_us.json

```diff
@@ -403,6 +403,8 @@
     "tag.doused":"🔥",
     "tag.dead":"💀",
 
+    "noAll.icon": "🔇",

```

Comment:
This one doesn't start with chatGroup like the rest

Reply:
thats because "noAll" isnt a chatgroup